packages/platform-sdk-profiles/src/dto/transaction.test.ts

```diff
@@ -587,6 +601,26 @@ describe("EntityRegistrationData", () => {
 	test("#ipfs", () => {
 		expect(subject.ipfs()).toBe("ipfs");
 	});
+
+	test("#ipfsContent for undefined ipfs", async () => {
+		subject = createSubject(wallet, {
+			...subject,
+			ipfs: () => undefined,
+		}, EntityRegistrationData)
+		expect(await subject.ipfsContent()).toBeUndefined();
+	});
+
+	test("#ipfsContent", async () => {
+		subject = createSubject(wallet, {
+			...subject,
+			ipfs: () => 'QmR45FmbVVrixReBwJkhEKde2qwHYaQzGxu4ZoDeswuF9c',
+		}, EntityRegistrationData)
+		expect(await subject.ipfsContent()).toBe("ipfs-content");
```

Comment:
`await expect(subject.ipfsContent()).resolves.toBe("ipfs-content");`

Always use `.resolves` or `.rejects` and `await` the `expect` to avoid false-positives or completely false results.

Reply:
:+1: I'll fix it in next branch